cfgov/jinja2/v1/_includes/macros/activity-snippets.html

```diff
@@ -78,14 +78,17 @@
       {% set icon = category_icon.render(activity_type) %}
       <div class="activity">
           <h3 class="h4">{{ icon }} {{ header }}</h3>
-          <ul class="list list__unstyled list__spaced list__links">
+          <ul class="m-list
+                     m-list__unstyled
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/113e79bec50e722f9e508acb0004c64b4cf4a511